src/helpers/trimPhotosResponse.ts

```diff
@@ -0,0 +1,14 @@
+import { PhotosResponse, TrimmedPhotosResponse } from "../models/PhotosResponse";
+
+const trimPhotosResponse = (photosResponse: PhotosResponse): TrimmedPhotosResponse => {
+    return {
+        photos: photosResponse.photos.map((photo) => {return {
+            id: photo.id,
+            sol: photo.sol,
+            earth_date: photo.earth_date,
+            img_src: photo.img_src
+        }})

```

Comment:
```suggestion
        photos: photosResponse.photos.map((photo) => ({
            id: photo.id,
            sol: photo.sol,
            earth_date: photo.earth_date,
            img_src: photo.img_src
        }))
```

you can use `(` `)` if you're just returning an object